src/components/CreateVmWizard/steps/Storage.js

```diff
@@ -362,6 +402,33 @@ class Storage extends React.Component {
     ]
   }
 
+  // return boolean value to answer if we are editing a Disk or not
+  isEditingMode = () => Object.keys(this.state.editing).length > 0
```

Comment:
Please convert this to the less cool function-bound-in-constructor style to maintain consistency in the component.